docs/CHUNKED_CHECKSUMS.md

```diff
@@ -34,7 +34,9 @@ as of at least [boto3 1.34.44](https://pypi.org/project/boto3/1.34.44/)
 * Prefix: 0xb510
 * Status: draft
 * Type: multihash
-* Description: Hash of concatenated SHA2-256 digests of 8*2^n MiB source chunks;
-  n = ceil(log2(source_size/(10^4 * 8MiB)))
-* Registrar: [multiformats/multicodec](https://github.com/multiformats/multicodec) 
-* Registration Date: [2024-02-23](https://github.com/multiformats/multicodec/pull/343)
\ No newline at end of file
+* Description: Hash of concatenated SHA2-256 digests of `8*2^n MiB` source chunks;
+`n = ceil(log2(source_size/(10^4 * 8MiB)))`
```

Comment:
I'm not sure using backticks here is justfied

Reply:
I don't understand why this whole section is here. But if it is there, you may be right, that it should be identical to https://github.com/multiformats/multicodec/pull/343/files

However, ` * ` is treated like the start of *emphasizing*, and, also, it is a formula, so, it can be marked as code.

Reply:
I'm disabling the failing lint rule here